pkg/deploymanager/config.go

```diff
@@ -0,0 +1,132 @@
+package deploymanager
+
+import (
+	"os"
+
+	olmclient "github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	lvmv1 "github.com/red-hat-storage/lvm-operator/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// InstallNamespace is the namespace lvm is installed into
+const InstallNamespace = "openshift-storage"
+
+// DeployManager is a util tool used by the tests
+type DeployManager struct {
+	olmClient      *olmclient.Clientset
+	k8sClient      *kubernetes.Clientset
+	crClient       crclient.Client
+	parameterCodec runtime.ParameterCodec
+	lvmClient      *rest.RESTClient
+}
+
+// GetCrClient is the function used to retrieve the controller-runtime client.
+func (t *DeployManager) GetCrClient() crclient.Client {
+	return t.crClient
+}
+
+// GetK8sClient is the function used to retrieve the kubernetes client.
+func (t *DeployManager) GetK8sClient() *kubernetes.Clientset {
+	return t.k8sClient
+}
+
+// GetParameterCodec is the function used to retrieve the parameterCodec.
+func (t *DeployManager) GetParameterCodec() runtime.ParameterCodec {
+	return t.parameterCodec
+}
+
+// GetLvmClient is the function used to retrieve the lvm client.
+func (t *DeployManager) GetLvmClient() *rest.RESTClient {
+	return t.lvmClient
+}
+
+func getKubeconfig(kubeconfig string) (error, *rest.Config) {
```

Comment:
Generally error is the last value returned. Please swap the locations.

Reply:
Updated.